src/github/nag.rs

```diff
@@ -883,7 +979,16 @@ enum CommentType<'a> {
         status_comment_id: i32,
         added_label: bool,
     },
-    FcpWeekPassed,
+    FcpWeekPassed {
+        author: &'a GitHubUser,
+        status_comment_id: i32,
+        added_label: bool,
+        disposition: FcpDisposition
+    },
+}
+
+fn is_rfc_repo(issue: &Issue) -> bool {
```

Comment:
Can you make this a part of `teams.toml` (maybe rename to `rfcbot.toml` and move current fields under a top-level teams key)? It would be very nice to not regress progress towards https://github.com/anp/rfcbot-rs/issues/92.